src/darwintracelib1.0/darwintrace.c

```diff
@@ -1015,5 +1181,23 @@ bool __darwintrace_is_in_sandbox(const char *path, int flags) {
 		}
 	} while (pathIsSymlink);
 
-	return __darwintrace_sandbox_check(normPath, flags);
+    //Check shared memory cache for path
```

Comment:
I will fix that before making the next commit for sure. Thanks for giving it a review :) Also, this became the case cuz I shifted `__darwintrace_setup()` calls a lot. `__darwintrace_setup()` were surprisingly the most time taking calls of all. Before the assumption was `__darwintrace_is_in_sandbox()`, when registry is checked, is the most time taking. So for that I only did setup on demand coz of which I ended up messing up the code. Also, I actually have conducted new tests with new changes which I had been discussing in mailing lists. I will update them by today night on here. Currently, I am testing on a rental Mac with HDD (previous tests were on ssd) now because I thought that’s where registry calls should actually be slow and that should have been the reason to come up with this project.